src/core/net/dns_types.hpp

```diff
@@ -1267,6 +1267,7 @@ class ResourceRecord
     // Resource Record Types.
     static constexpr uint16_t kTypeZero  = 0;   ///< Zero as special indicator for the SIG RR (SIG(0) from RFC 2931).
     static constexpr uint16_t kTypeA     = 1;   ///< Address record (IPv4).
+    static constexpr uint16_t kTypeNs    = 2;   ///< NS record (an authoritative name server)
```

Comment:
```suggestion
    static constexpr uint16_t kTypeNs    = 2;   ///< NS record (an authoritative name server).
```

Reply:
Thanks. All fixed in the new pushed commit.